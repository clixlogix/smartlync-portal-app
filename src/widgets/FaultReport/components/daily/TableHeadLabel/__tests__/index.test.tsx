import * as React from 'react';
import { render } from '@testing-library/react';

import { TableHeadLabel } from '..';

describe('<TableHeadLabel  />', () => {
    it('should match snapshot', () => {
        const loadingIndicator = render(<TableHeadLabel />);
        expect(loadingIndicator.container.firstChild).toMatchSnapshot();
    });
});
