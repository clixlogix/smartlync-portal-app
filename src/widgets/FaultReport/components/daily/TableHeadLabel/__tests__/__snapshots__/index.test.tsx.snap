// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableHeadLabel  /> should match snapshot 1`] = `
<div
  class=" x-cls-table-head-label"
>
  <div
    class=""
  />
</div>
`;
